src/Microsoft.Health.Fhir.SqlServer/Features/Schema/ISchemaDataStore.cs

```diff
@@ -24,5 +24,29 @@ public interface ISchemaDataStore
         /// /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns>The current schema versions information</returns>
         Task<GetCurrentVersionResponse> GetCurrentVersionAsync(CancellationToken cancellationToken);
+
+        /// <summary>
+        /// Delete expired instance schema information.
+        /// </summary>
+        /// <returns>A task</returns>
+        Task DeleteExpiredRecords();
+
+        /// <summary>
+        /// Upsert current version information.
+        /// </summary>
+        /// /// <param name="name">The instance name.</param>
+        /// /// <param name="schemaInformation">The Schema information</param>
+        /// /// <param name="cancellationToken">The cancellation token.</param>
```

Comment:
Nit: looks like all the `param` comments are accidentally commented twice in this file.